bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -259,6 +263,27 @@ public BigtableAsyncAdmin(CommonConnection asyncConnection) throws IOException {
     return listTables(Optional.of(pattern));
   }
 
+  @Override
+  public CompletableFuture<List<TableDescriptor>> listTableDescriptors(List<TableName> list) {
```

Comment:
Please add an integration test for testing null & empty list cases and run them using the minicluster profile and bigtable profile to make sure that the beahvior matches

Reply:
Also check if the returned list should contain column families